platforms/android/tangram/src/main/java/com/mapzen/tangram/viewholder/GLViewHolder.java

```diff
@@ -67,7 +68,8 @@
 
     /**
      * Can be used by the client to get access to the underlying view to pass any view controls
-     * @return {@link GLSurfaceView} or Client provided implementation for a view held by {@link GLViewHolder}
+     * @return {@link GLSurfaceView} or {@link TextureView} or Client provided implementation for
```

Comment:
No need to call out (or import) specific implementations, it's just a `GLSurfaceView`